definitions/json/CaseEventToFields.json

```diff
@@ -3469,6 +3469,26 @@
     "PageLabel": "Final notes",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
```

Comment:
do we need to show these fields in the event? surely these would be auto populated on submission?

Reply:
removed it